Packages/ClientRuntime/Sources/Waiters/Acceptor.swift

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// `Acceptor` is a Swift-native equivalent of Smithy acceptors:
+/// https://awslabs.github.io/smithy/2.0/additional-specs/waiters.html#acceptor-structure
+public struct Acceptor<Input, Output> {
```

Comment:
Nit: It'd be helpful to provide a description as to what an _Acceptor_ is or what is does